arlas-rest/src/main/java/io/arlas/server/rest/explore/search/GeoSearchRESTService.java

```diff
@@ -530,4 +439,23 @@ protected FeatureCollection getFeatures(CollectionReference collectionReference,
         }
         return fc;
     }
+
+    private Response search(CollectionReference collectionReference, Filter filter, String partitionFilter,
```

Comment:
Is it possible to rename it to `geosearch` as you did for `geoaggregate` in GeoAggregateRESTSevice ?

Reply:
Done